### PR TITLE
Unwire the legacy router

### PR DESCRIPTION
**Pull request by p0mvn**
## What is the purpose of the change:

- Unwire the legacy router
- Always return default Sidecar router
- Remove indexer pools
- Remove old feature flags

Next PR:
- Remove the actual implementation of the legacy router

### CODE REVIEW

File: packages/server/src/queries/complex/swap-routers.ts
Comment (jonator) [inline packages/server/src/queries/complex/swap-routers.ts:1]: IMO, we should go further and just call the SQS out-given-in query function and not even conform to the `TokenOutGivenInRouter` interface. We should be able to delete the pools package entirely.

**Review by jonator — APPROVED** [review]

File: packages/server/src/queries/complex/swap-routers.ts
Comment (p0mvn) [reply packages/server/src/queries/complex/swap-routers.ts:1]: OK I will try

**Review by kamal-sutra — APPROVED** [review]

**Review by jonator — APPROVED** [review]
This is a good cleanup step.

Without spending tons of time digging into the code, I think the next step to evetually delete the pools package (or help us be ready to) would be to copy over any imports directly from the pools package into /packages/server/src/queries/sidecar/router.ts (like error types, the router interface, etc).

Basically removing references into the pools package. Then when the tRPC migration is complete and all legacy pools queries are deleted, we can just delete that package in entirety.

**Comment by p0mvn** [comment]
> This is a good cleanup step.
> 
> Without spending tons of time digging into the code, I think the next step to evetually delete the pools package (or help us be ready to) would be to copy over any imports directly from the pools package into /packages/server/src/queries/sidecar/router.ts (like error types, the router interface, etc).
> 
> Basically removing references into the pools package. Then when the tRPC migration is complete and all legacy pools queries are deleted, we can just delete that package in entirety.

Thanks for suggestions - tracked here: https://github.com/osmosis-labs/osmosis-frontend/issues/3764

**Comment by p0mvn** [comment]
Merging - appreciate the reviews

**Pull request merged**